src/components/TopBar/components/SearchField/SearchField.scss

```diff
@@ -94,7 +114,8 @@ $stacking-order: (
 }
 
 .Clear {
-  @include recolor-icon(var(--p-action-secondary, color('ink', 'lightest')));
+  @include recolor-icon(var(--p-icon, color('ink', 'lightest')));
+  @include focus-ring($size: 'wide');
```

Comment:
Can we apply the focus ring to the icon within the button, instead of the button itself? I think that will make it look less like the ring is floating in space around the icon.